sqe-api-server/Helpers/InputValidation.cs

```diff
@@ -106,14 +109,17 @@ private static string _validatePolygon(string wktPolygon, string entityName, boo
             if (string.IsNullOrEmpty(wktPolygon))
                 throw new StandardExceptions.InputDataRuleViolationException("The submitted WKT POLYGON is empty");
 
-            var polygon = default(Geometry);
             // Try loading the polygon
             try
             {
-                polygon = wkr.Read(wktPolygon);
+                var polygon = wkr.Read(wktPolygon);
                 // If it is valid, return it
                 if (polygon.IsValid)
-                    return polygon.Normalized().ToString(); // Always normalize the polygon
+                {
+                    // Remove any completely unnecessary points
+                    var simplifier = new DouglasPeuckerSimplifier(polygon) { DistanceTolerance = 0 };
```

Comment:
This code still uses the rust code, but it also makes a pass to remove any unnecessary point (e.g., a line with three point 0 0, 10 0, 20 0 gets simplified to two points 0 0, 20 0).  Its not really part of the polygon validation, just a little something to save some space since polygons from the front end often have unnecessary points.